demo/src/app/getting-started/getting-started.component.html

```diff
@@ -1,5 +1,15 @@
 <ngbd-content-wrapper component="Getting Started">
   <h3>
+    Current State and Status
+  </h3>
+  <p>
+    <a href="https://badge.fury.io/js/%40ng-bootstrap%2Fng-bootstrap"><img src="https://badge.fury.io/js/%40ng-bootstrap%2Fng-bootstrap.svg" alt="npm version" height="18"></a>
+    <a href="https://travis-ci.org/ng-bootstrap/ng-bootstrap.svg?branch=master"><img src="https://travis-ci.org/ng-bootstrap/ng-bootstrap.svg?branch=master"/></a>
+    <a href="https://david-dm.org/ng-bootstrap/ng-bootstrap" title="dependencies status"><img src="https://david-dm.org/ng-bootstrap/ng-bootstrap/status.svg"/></a>
+    <a href="http://issuestats.com/github/ng-bootstrap/ng-bootstrap"><img src="http://issuestats.com/github/ng-bootstrap/ng-bootstrap/badge/pr"/></a>
+    <a href="http://issuestats.com/github/ng-bootstrap/ng-bootstrap"><img src="http://issuestats.com/github/ng-bootstrap/ng-bootstrap/badge/issue"/></a>
+  </p>
+  <h3>
```

Comment:
I don't think it makes sense to add it to the "Getting started" page (BTW, we should probably rename it to "Installation").

IMO the role of this page is to get people up & running quickly with minimal number of distractions.